session_0/fizz_buzz.rb

```diff
@@ -37,5 +37,23 @@
 # We will raise an `ArgumentError` exception to let the caller know that
 # their function arguments were incorrect.
 def fizz_buzz(n:, x: 3, y: 5)
-  raise NotImplementedError # TODO
+  values=Array.new()
```

Comment:
Additionally, you should enable github actions - so that tests are run automatically when you push code.

1. Go to your repository
2. Go to actions
3. Enable the workflows

![Screenshot_2021-01-04 abhishekkumar2718 IRIS-RoR-Bootcamp-2020](https://user-images.githubusercontent.com/31231064/103504307-09f71b80-4e7d-11eb-9ace-f1b6cf920fb9.png)